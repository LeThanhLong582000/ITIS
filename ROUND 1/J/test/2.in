91
467452392 666474247
451559118 70290407
954330232 100523396
584135842 941005559
7891294 566989108
990045751 911883653
154937231 855088327
931384817 134690473
30969781 644650506
727678194 785489722
934863520 311059999
519436211 17689691
203825608 11116171
152478867 496631394
544910044 458108281
234962128 71130039
743410666 594886439
68531491 531039107
821090055 256942438
452937828 757821929
974002263 956675127
357248657 697425233
584547617 60208457
842352130 459119448
683782256 242778350
620930394 660576410
71783624 862640649
819263052 579558347
960773906 670843702
198375415 528937310
883831672 232214316
812704296 12050410
370866625 909921896
306776724 363318326
321589741 142477675
636617399 430348090
864910257 634333326
531534514 121153405
671194412 454715422
10797932 874621574
640512355 298046002
187151648 170605912
277241104 981438638
850678820 698834333
422147301 173255881
106693881 260283056
632263585 538244252
793147009 774740087
519661144 784318066
87429530 840639570
978447938 168696895
235135541 836850667
538927531 161163069
321422978 937282897
281277829 762642473
910067075 181355253
73669788 224368887
887778935 728882269
861577585 901917442
761372764 425604809
295739170 817568025
322855042 795682620
677263288 169173008
958615039 690147155
305130397 638064847
968980556 596492014
450273655 503337794
107136911 662284105
812327690 280870767
625187758 274399302
58401442 141130256
698390563 548311615
726282288 133058236
790928951 740815498
761501280 470632324
615582418 794420117
340832350 39928513
184438094 54524242
790259274 3624392
385449231 814408649
978410050 503100817
129176613 649940123
516703606 861169779
813420369 350075783
723984358 514525291
739616884 152920388
49281794 61156929
870255897 362001252
261339107 29330197
182312057 1242311
467805872 327274383
