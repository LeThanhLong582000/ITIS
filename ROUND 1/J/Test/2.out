3739619136
3160913826
7634641856
4673086736
63130352
7920366008
1239497848
7451078536
247758248
5821425552
7478908160
3636053477
1426779256
1219830936
4359280352
1644734896
5947285328
548251928
6568720440
3623502624
7792018104
2857989256
4091833319
6738817040
5470258048
4967443152
574268992
6554104416
7686191248
1587003320
7070653376
5688930072
2966933000
2454213792
2572717928
5092939192
6919282056
4252276112
5369555296
86383456
5124098840
1497213184
2217928832
6805430560
3377178408
853551048
5058108680
6345176072
4157289152
699436240
7827583504
1881084328
4311420248
2571383824
2250222632
7280536600
589358304
7102231480
6892620680
6090982112
2365913360
2582840336
5418106304
7668920312
2441043176
7751844448
3602189240
857095288
6498621520
5001502064
467211536
5587124504
5810258304
6327431608
6092010240
4924659344
2385826450
1291066658
4741555644
3083593848
7827280400
1033412904
4133628848
6507362952
5791874864
5916935072
344972558
6962047176
1829373749
1093872342
3742446976
